en/orm/table-objects.rst

```diff
@@ -166,17 +166,17 @@ buildRules
 
 .. php:method:: buildRules(Event $event, RulesChecker $rules)
 
-The ``Model.buildRules`` event is fired before after a rules instance has been
-created and the table's ``beforeRules()`` method has been called.
+The ``Model.buildRules`` event is fired after a rules instance has been
+created and before the table's ``beforeRules()`` method has been called.
```

Comment:
The order in the code is:

1. Table::buildRules() is called.
2. Model.buildRules event is triggered.
3. Model.beforeRules event is triggered.
4. Check is done
5. Model.afterRules event is triggered.